lib/ship.rb

```diff
@@ -11,10 +11,6 @@ def initialize(name, length)
     @health = length
   end
 
-  def health
-    @health
- end
```

Comment:
Looking over my notes (from class and Mythical Creatures) it seems like having `:health` in the `attr_reader` accomplishes exactly the same thing as this method, so I removed the method to see what would happen and all the tests passed. If you want to add it back for safe keeping, no problem.

Reply:
Im all for getting rid of redundant code! I don't know how I didnt notice haha